core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -248,6 +248,10 @@ object ConsoleConsumer extends Logging {
       .withRequiredArg
       .describedAs("prop")
       .ofType(classOf[String])
+    val messageFormatterConfigOpt = parser.accepts("config", s"Config properties file to initialize the message formatter. Note that $messageFormatterArgOpt takes precedence over this config.")
```

Comment:
We already have an --consumer.config argument for properties file in `ConsoleConsumer`

Reply:
`--consumer.config` is to setup the consumer, this is to setup the message formatter.
It the same difference as `--consumer-property` and `--property`.